scio-extra/src/main/scala/com/spotify/scio/extra/checkpoint/package.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.extra
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.io.FileStorage
+import com.spotify.scio.testing.ObjectFileIO
+import com.spotify.scio.util.ScioUtil
+import com.spotify.scio.values.SCollection
+
+import scala.reflect.ClassTag
+
+/**
+ * Main package for checkpoint API. Import all.
+ *
+ * {{{
+ * import com.spotify.scio.extra.checkpoint._
+ * }}}
+ */
+package object checkpoint {
+
+  // scalastyle:off method.name
+  /**
+   * For use in testing, see [[CheckpointExampleTest]].
+   */
+  def CheckpointIO[T](fileOrPath: String): ObjectFileIO[T] = ObjectFileIO[T](fileOrPath)
```

Comment:
Would probably not buy us much given that there are two paths for saveAsPB - one for testing with protobuf objects and other after conversion to avro. In this case checkpoints use object IO for both paths, so had to do this trick.